docs/source/toc.yaml

```diff
@@ -52,5 +52,24 @@ subtrees:
                     title: Deploy the HugeCTR Model with Triton
                   - file: examples/scaling-criteo/04-Triton-Inference-with-Merlin-Models-TensorFlow.ipynb
                     title: Deploy the TensorFlow Model with Triton
+      - file: api_overview
+        title: API Documentation
```

Comment:
The docs become a little dense with this all under one TOC item.  If you are open to experimenting, consider creating another `caption` like line 3 with "API Documentation" or "Merlin API" or something along those lines.

Centralizing the API is certainly better than how I left it, but I'm also wondering if adding titles would help the reader choose the correct API rather than browse.

Reply:
I feel the structure looks easy to follow - it is structured in the way how we would import Classes, functions or modules

```
merlin.dag
merlin.loader
merlin.io
merlin.models
merlin.schema
merlin.systems
nvtabular
```

Reply:
It might still be nice to organize the packages into themes or something though, because I suspect this makes the most sense to people who are already familiar with what's in each sub-package (e.g. people on the Merlin team.)

Reply:
@karlhigley do you have a suggestions? I thought organizing it by import statements is the way a user will search for API documentation?

If I want to know something about ColumnsSelector, I know it is in merlin.schema and I will look there first?